manager/controllers/app/m4dapplication_controller.go

```diff
@@ -139,6 +140,31 @@ func ContainsFinalizer(obj *app.M4DApplication, finalizer string) bool {
 	return false
 }
 
+// DeleteExternalResources deletes all resources allocated by the controller: blueprint, storage, namespace
+func (r *M4DApplicationReconciler) DeleteExternalResources(applicationContext *app.M4DApplication) error {
```

Comment:
I think that this function don't have to be exported

Reply:
OK